db/fixtures/ae_datastore/ManageIQ/GenericObject/Example.class/__class__.yaml

```diff
@@ -0,0 +1,93 @@
+---
```

Comment:
@lfu The automate modeling should not be part of this PR, let's keep it focused on the backend code.  Also, the export I created was meant for initial testing.  We also do not want to include the `LoadBalancer` namespace in the final export.